website/blog/2021-07-29-Introduction to Flint.md

```diff
@@ -0,0 +1,92 @@
+---
+slug: intro to flint
+title: Introduction to FLINT
+author: Ruchi Pakhle
+author_url: https://github.com/Ruchip16
+author_title: Passionate programmer, coder & a web-developer 
+tags: [Intro to Flint, docusaurus]
+description: Introduction to Flint
+---
+<!-- Summary -->
+
+**The Full Lands Integration Tool (FLINT)** is moja global’s flagship software. FLINT is a modular and highly flexible open-source software technology designed to meet the needs of governments, the private sector and other users in measuring, reporting and verifying (MRV) – or forecasting – greenhouse gas emissions and removals from forestry, agriculture and other land uses (AFOLU).
```

Comment:
This section is directly adopted from the website [here](https://moja.global/flint/). Please re-word.